recipes/jupyterlab-topbar/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "jupyterlab-topbar" %}
+{% set version = "0.6.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab-topbar-{{ version }}.tar.gz
+  sha256: d8aa3b377c885c4732cc487dd74baf376502142cf89025e177ef33c733f8c56f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - jupyter-packaging
+    - jupyterlab >=3
+    - pip
+    - python >=3.6
+    - nodejs >=12
```

Normally `jupyterlab` and `nodejs` shouldn't be needed anymore (prebuilt assets in the sdist):

```suggestion
    - pip
    - python >=3.6
```